Carsino - Carga Docente terminado

TheCarsino opened the pull request:
Carga Docente finalizado

Falta Carga Masiva en CRUD Docentes - COULD

Aaron123098 committed:
inicio cargadocentecursos

Aaron123098 committed:
Merge remote-tracking branch 'origin/gabrhr' into aaroncillo

Aaron123098 committed:
Avance 2 ad

Aaron123098 committed:
Avance 3 ad

Aaron123098 committed:
Merge remote-tracking branch 'origin/Carsino' into aaroncillo

Aaron123098 committed:
Asignar curso a ciclo casi listo

faltan los disableds y el post

MBecerra777 committed:
Se intenta agregar carga masiva de docentes

zzz

MBecerra777 committed:
Merge branch 'develop' into marcelo

TheCarsino committed:
Merge remote-tracking branch 'origin/aaroncillo' into Carsino

MBecerra777 committed:
Sí que funciona el eliminar Docentes XD

F

TheCarsino committed:
Carga Docente actualizada con nueva data del Backend

La funcionalidad está igual a la que se tenía el Martes. Solo que con las nuevas pantallas de Docente y Curso CRUD

MBecerra777 committed:
Merge branch 'develop' into marcelo

TheCarsino committed:
Merge remote-tracking branch 'origin/marcelo' into Carsino

zazke commented:
- carga masiva docentes (x)
- carga masiva cursos (x)
- Nuevo Curso de Asignacion de Curso (x)
- Carga Horaria de profesores no recarga automaticamente.  Boton Send no hace nada. (x)
- NavBar mesa de partes (x)

zazke committed:
Quitar MesaPartes del NavBar de AS

TheCarsino committed:
Carga Docente Asistente de Sección - Version preliminar

Hace falta que se actualizen los campos de docentes asignados a las sesiones de los horarios. Además de los cambios que nos presentó Chung.

TheCarsino committed:
Merge branch 'Carsino' of https://github.com/gabrhr/teachersoftJS into Carsino

TheCarsino committed:
Merge branch 'develop' into Carsino

TheCarsino commented:
ACTUALIZADO CON LOS ULTIMOS CAMBIOS

erickOz commented:
Revisanding

erickOz reviewed (approved):
Cambios Anteriores 
Falta Validaciones Excel Import

* *NAVBAR : ASIGNACION DE CARGA* **
-REGISTRO DE CURSOS: Gestión de la carga de Horarios
Cuando termino de Registrar un Horario para Curso me debería regresar a la pantalla de Horarios y no mantenerme en la pantalla de Creación de Nuevo Horario.
Cuando se desea CANCELAR la creación un nuevo Horario para un Curso NO pasa nada
Descargar Plantilla OK
Delimitar no aceptar PDF's en subida de cursos
Error Subida de Curso con Plantilla - Archivo: ModalAsignacionCarga.js revisar
Exportar Plantilla OK
--
-REGISTRAR CARGA: Registro de Carga Docente
Muestra lista de Cursos
No funciona el FILTRO POR ESTADOS
Visualizar detalle de Curso OK
No permite asignar un profesor ni a Clase ni Laboratorio (boton Guardar no funciona no muestra feedback)
--
-DEUDA Y DESCARGA: Gestión de Deudas y Desacargas Horarios
Lista de Docentes con info OK
Al revisar Pagina Deudas y Descargas - Parpadean las fotos de Cueva, khlebni y  Flores (solo las de ellos)
--

**NAVBAR : DOCENTES**
Lista Docentes CRUD OK
**NAVBAR: CURSOS**
Cursos CRUD OK
Asignación de Cursos para ser Dictados en el Ciclo OK
Falta Plantilla para poder Importar Cursos Masivamente
